packages/block-editor/src/components/inserter/media-tab/utils.js

```diff
@@ -27,10 +31,16 @@ export function getBlockAndPreviewFromMedia( media, mediaType ) {
 	const PreviewTag = mediaTypeTag[ mediaType ];
 	const preview = (
 		<PreviewTag
-			src={ mediaSrc }
+			src={ media.previewUrl || mediaSrc }
 			alt={ alt }
 			controls={ mediaType === 'audio' ? true : undefined }
 			inert="true"
+			onError={ ( { currentTarget } ) => {
+				// Fallback to the media source if the preview cannot be loaded.
```

Comment:
```suggestion
				// Fall back to the media source if the preview cannot be loaded.
```

([English phrasal verbs](https://en.wikipedia.org/wiki/English_phrasal_verbs) are kept split, while nouns/adjectives tend to use hyphens when the term is new, and ultimately progress to single-word form as they become commonplace)